components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -134,7 +143,13 @@
 <config_use_GM ocn_grid="oRRS15to5">.false.</config_use_GM>
 <config_GM_kappa>1800.0</config_GM_kappa>
 <config_GM_kappa ocn_forcing="datm_forced_restoring" ocn_grid="oEC60to30v3wLI">600.0</config_GM_kappa>
-<config_GM_closure>'N2_dependent'</config_GM_closure>
+<config_GM_closure>'visbeck'</config_GM_closure>
+<config_GM_baroclinic_mode>3.0</config_GM_baroclinic_mode>
+<config_GM_visbeck_alpha>0.005</config_GM_visbeck_alpha>
+<config_GM_visbeck_min_depth>50.0</config_GM_visbeck_min_depth>
+<config_GM_visbeck_max_depth>1000.0</config_GM_visbeck_max_depth>
+<config_GM_visbeck_min_kappa>300.0</config_GM_visbeck_min_kappa>
+<config_GM_visbeck_max_kappa>1200.0</config_GM_visbeck_max_kappa>
```

Comment:
@vanroekel you specified `config_GM_visbeck_max_kappa = 1200.` on your post above, but please double check these other three for depth and kappa, which just came from the defaults.